rootless.md

```diff
@@ -14,6 +14,9 @@ can easily fail
 * No cgroup Support (hopefully fixed when cgroups V2 happens).
   * Cgroups V1 does not safely support cgroup delegation.
   * Cgroups V2 development for container support is ongoing.
+* Some systemd's unit configuration options does not work
```

Comment:
We should be clear that this refers to systemd in a container, not as PID1

Reply:
Hmm, but systemd is a PID1 in a container.

I'd be happy to add "... does not work in the rootless container." if you insist. However, I was assuming that the context of the rootless container is implied.

Also, I'd not expect that by using rootless containers I'd lose the ability to use cgroupsv1. IOW, the paragraph above seems to make the same assumption.